bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx

```diff
@@ -247,7 +248,7 @@ class UserListItem extends Component {
 
     const userNameSub = [];
 
-    if (user.isLocked) {
+    if (isMeetingLocked && !user.isPresenter && !user.isModerator) {
```

Comment:
You have checked for `isViewer` rather than `!user.isPresenter && !user.isModerator` in other spots in the PR. I'd recommend keeping it consistent